### ISSUE TITLE
Unable to install from pip

### ISSUE THREAD

**Issue by pconwell** [jasonrollins/shareplum2#2]
`$ pip install shareplum2`

```
ERROR: Could not find a version that satisfies the requirement shareplum2 (from versions: none)
ERROR: No matching distribution found for shareplum2
```

And with `pipenv`:
```
ERROR: ERROR: Could not find a version that matches shareplum2
No versions found
Was https://pypi.org/simple reachable?
```

I'm assuming this is just because the package hasn't been published yet?

**Comment by jasonrollins** [comment]
Yes that correct. Sorry for that. I'm pushing some changes soon that might
break things. So I'm sort of deciding what to do.

**Comment by pconwell** [comment]
I kinda figured that's what it was. I just wanted to make sure.

Thanks!

**Comment by ejyoung82** [comment]
Is installing this from pip still down?  I just tried with no success.  We are being forced to start using the 365 sites soon so I was hoping to get a jump on converting.

**Comment by jasonrollins** [comment]
Yeah ignore this repository. I'm going to put everything I planned to do
here back in shareplum and delete this one
